sky/templates/spot-controller.yaml.j2

```diff
@@ -33,6 +33,10 @@ setup: |
     source ~/google-cloud-sdk/path.bash.inc > /dev/null 2>&1 && \
     echo 'source ~/google-cloud-sdk/path.bash.inc > /dev/null 2>&1' >> ~/.bashrc)
 
+  {% if is_dev %}
+  echo 'export SKYPILOT_DEV=1' >> ~/.bashrc
+  {% endif %}
```

Comment:
Just curious, why is this needed? Our `envs` section below should cover all the `setup` and `run` commands on the controller. Is it for logging into the controller through ssh and debugging?

Reply:
Exactly for that use case; added a comment.